recipes/tvb-framework/meta.yaml

```diff
@@ -0,0 +1,67 @@
+
+package:
+  name: tvb-framework
+  version: "1.5.4"
+
+source:
+  git_rev: 1aaf90c67450dfe535a7f76002f203218de330b4
+  git_url: https://github.com/the-virtual-brain/tvb-framework.git
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install
+  skip: True  # [py>27]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - allensdk
```

Comment:
These are all needed as `run` dependencies, but actually _not_ needed as build dependencies once you add `--single-version-externally-managed --record record.txt`.